### ISSUE TITLE
Difference between this repo(Chimpy) and [Chimp](https://github.com/xolvio/chimp)?

### ISSUE THREAD

**Issue by ozzywalsh** [TheBrainFamily/chimpy#124]
Difference between this repo(Chimpy) and [Chimp](https://github.com/xolvio/chimp)?

Some more clarity on this is needed,

**Comment by abecks** [comment]
I'm under the impression that "Chimpy" is the repo we know and love and is what they mean by "Chimp 1.0". The xolvio Chimp repo is going to be completely rewritten as "Chimp 2.0", and won't receive bug fixes for the 1.0 "Chimpy" line.

**Comment by lgandecki** [comment]
Yeah, @abecks is right. How did you guys even find this repo out so quickly? We are looking to make a proper announcement this weekend.

**Comment by abecks** [comment]
@lgandecki Chimp announced it on their website: https://chimp.readme.io/

**Comment by lgandecki** [comment]
well, that's definitely a good point :D
